pipelines/go/bump.yaml

```
@@ -36,8 +35,14 @@ inputs:
 
 pipeline:
   - runs: |
-      cd "${{inputs.modroot}}"
+      deps="${{inputs.deps}}"
+      replaces=${{inputs.replaces}}
```

```suggestion
      replaces="${{inputs.replaces}}"
```

For consistency I'd propose quoting both the same way - safer this way.